causal_testing/testing/estimators.py

```diff
@@ -352,7 +352,14 @@ def estimate_coefficient(self) -> tuple[pd.Series, list[pd.Series, pd.Series]]:
         model = self._run_linear_regression()
         newline = "\n"
         patsy_md = ModelDesc.from_formula(self.treatment)
-        if any((self.df.dtypes[factor.name()] == 'object' for factor in patsy_md.rhs_termlist[1].factors)):
+        if any(
```

Comment:
@jmafoster1 For my understanding, does this avoid the errors because you're checking if the terms already exist in the original dataframe? I'm happy to pass this for now, but can you send me a simple interaction term example when you have time so I can look into this?

Reply:
Yes, that's right. Thanks for approving. I will be publishing my replication package in a minute, which has the example in, so I'll send you a link